pkg/server/container_status.go

```diff
@@ -88,3 +93,38 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 		LogPath:     meta.LogPath,
 	}
 }
+
+// toCRIContainerInfo converts internal container object information to CRI container status response info map.
+func toCRIContainerStatusResponseInfo(ctx context.Context, container containerstore.Container, verbose bool) map[string]string {
+	var info map[string]string
+
+	if verbose {
+		info = make(map[string]string)
+		meta := container.Metadata
+		status := container.Status.Get()
+
+		info["pid"] = marshallToString(status.Pid)
+		info["container config"] = marshallToString(meta.Config)
+		info["removing state"] = marshallToString(status.Removing)
+
+		oldSpec, err := container.Container.Spec(ctx)
+		if err == nil {
+			info["container runtime spec"] = marshallToString(oldSpec)
+		}
+
+		ctrInfo, err := container.Container.Info(ctx)
+		if err == nil {
+			info["snapshotkey"] = marshallToString(ctrInfo.SnapshotKey)
```

Comment:
s/snapshotkey/snapshotKey
Let's use the golang json field naming convention.
`crictl info` is following the same convention, let's make them consistent.

Reply:
Done and I'm adding a todo to make them consistent by adding them as constants in CRI.

Reply:
That'll be hard. This constants may not make sense to other container runtimes, e.g. rkt, frakti. So it's hard to define this at CRI level.